gen_cert.py

```diff
@@ -80,36 +81,23 @@
 }
 
 
-def prettify_isodate(isoformat_date):
-    """Convert a string like '2012-02-02' to one like 'February 2nd, 2012'"""
-    m = RE_ISODATES.match(isoformat_date)
-    if not m:
-        raise TypeError("prettify_isodate called with incorrect date format: %s" % isoformat_date)
-    day_suffixes = {'1': 'st', '2': 'nd', '3': 'rd', '21': 'st', '22': 'nd', '23': 'rd', '31': 'st'}
-    months = ['January', 'February', 'March', 'April', 'May', 'June', 'July',
-              'August', 'September', 'October', 'November', 'December']
-    date = {'year': '', 'month': '', 'day': '', 'suffix': 'th'}
-    date['year'] = m.group('year')
-    date['month'] = months[int(m.group('month')) - 1]
-    date['day'] = m.group('day').lstrip('0')
-    date['suffix'] = day_suffixes.get(date['day'], 'th')
-    return "%(month)s %(day)s%(suffix)s, %(year)s" % date
-
-
-def get_cert_date(calling_date_parameter, configured_date_parameter):
+def get_cert_date(calling_date_parameter, configured_date_parameter, date_format='%B %d, %Y', locale="en_US"):
```

Comment:
I don't think we want to (or even should?) be manually specifying date formats; we can defer this to a date lib like `Babel`, locale dependent.